packages/braid-design-system/src/lib/components/Toggle/Toggle.tsx

```diff
@@ -26,6 +26,7 @@ export interface ToggleProps {
   align?: 'left' | 'right' | 'justify';
   togglePosition?: 'leading' | 'trailing';
   size?: Size;
+  /** @deprecated `bleedY` is standard, and configuration will be removed in a future version */
```

Comment:
Opting for consistency of language here, we use "enabled by default" everywhere else.

```suggestion
  /** @deprecated `bleedY` is enabled by default, and configuration will be removed in a future version */
```